nginx/tests/test_e2e.py

```diff
@@ -0,0 +1,57 @@
+# (C) Datadog, Inc. 2019
+# All rights reserved
+# Licensed under Simplified BSD License (see LICENSE)
+import pytest
+
+from datadog_checks.nginx import Nginx
+
+from . import common
+
+
+@pytest.mark.e2e
+def test_e2e(dd_agent_check, instance, instance_vts):
+    if common.USING_VTS:
+        instance = instance_vts
+    aggregator = dd_agent_check(instance, rate=True)
+
+    aggregator.assert_metric('nginx.net.writing', count=2, tags=common.TAGS)
+    aggregator.assert_metric('nginx.net.waiting', count=2, tags=common.TAGS)
+    aggregator.assert_metric('nginx.net.reading', count=2, tags=common.TAGS)
+    aggregator.assert_metric('nginx.net.conn_dropped_per_s', count=1, tags=common.TAGS)
+    aggregator.assert_metric('nginx.net.conn_opened_per_s', count=1, tags=common.TAGS)
+    aggregator.assert_metric('nginx.net.request_per_s', count=1, tags=common.TAGS)
+
+    if common.USING_VTS:
```

Comment:
IMHO we should avoid conditionals in tests. For unit tests we already use pytest `skipif` https://github.com/DataDog/integrations-core/blob/a92f42cb066fe99ff4d2716db2ce1e3b97e72873/nginx/tests/test_nginx.py#L9 

Maybe we can do the same for e2e.